app/src/test/java/com/kickstarter/viewmodels/RewardCardViewHolderViewModelTest.kt

```diff
@@ -67,9 +119,53 @@ class RewardCardViewHolderViewModelTest : KSRobolectricTestCase() {
 
         val creditCard = StoredCardFactory.discoverCard()
 
-        this.vm.inputs.configureWith(creditCard)
+        this.vm.inputs.configureWith(Pair(creditCard, ProjectFactory.project()))
 
         this.lastFour.assertValue("1234")
     }
 
+    @Test
+    fun testNotAvailableCopyIsVisible() {
```

Comment:
Here I would rename to `testNotAvailableCopyIsVisible_isFalse_whenCardIsAccepted`